tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -3743,6 +3872,12 @@ def calculateStagger(self, kernel, tP):
           imod.add(self.incrementSrd(tP["tpsMetadata"], sgpr(staggerTmp), sgpr(staggerTmp+1)))
 
     if tP["isB"]:
+      isDTVAorB = (not (kernel["DirectToVgprA"] and kernel["DirectToVgprB"])) and (kernel["DirectToVgprA"] or kernel["DirectToVgprB"])
```

Comment:
kernel["DirectToVgprA"] != kernel["DirectToVgprB"]

Reply:
I changed the if condition as you suggested (2 locations).